### PR TITLE
LAU-56: Add LAU Flux Kustomize pipeline configuration

### PR DESCRIPTION
**Pull request by bradley-hmcts**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/LAU-56

### Change description ###

Add LAU Flux pipeline configuration.

Only AAT and Demo are configured initially.

Managed identity configuration is still outstanding but will be added when LAU shared infrastructure is configured.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: k8s/namespaces/lau/lau-case-backend/lau-case-backend.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  name: lau-case-backend
+spec:
+  releaseName: lau-case-backend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/lau-case-backend
+  values:
+    java:
+      memoryRequests: "1024Mi"
```
Comment (timja) [inline k8s/namespaces/lau/lau-case-backend/lau-case-backend.yaml:13]: have you tested that this is necessary?

overriding defaults should only be done if proven via performance testing

File: k8s/namespaces/lau/lau-case-frontend/lau-case-frontend.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+  name: lau-case-frontend
+spec:
+  releaseName: lau-case-frontend
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/lau-case-frontend
+  values:
+    nodejs:
+      memoryRequests: "1024Mi"
```
Comment (timja) [inline k8s/namespaces/lau/lau-case-frontend/lau-case-frontend.yaml:13]: is this needed? frontend apps normally don't need to override the defaults unless they are large projects

**Review by timja — COMMENTED** [review]
can you look at my comments please

File: k8s/namespaces/lau/lau-case-backend/lau-case-backend.yaml (hunk 1 above)
Comment (bradley-hmcts) [reply k8s/namespaces/lau/lau-case-backend/lau-case-backend.yaml]: I've removed these overrides. Copied what we did with PCQ as the apps are of similar size.

File: k8s/namespaces/lau/lau-case-frontend/lau-case-frontend.yaml (hunk 2 above)
Comment (bradley-hmcts) [reply k8s/namespaces/lau/lau-case-frontend/lau-case-frontend.yaml]: I've removed these overrides. Copied what we did with PCQ as the apps are of similar size.

**Review by timja — APPROVED** [review]

**Pull request merged**